model/user.model.js

```diff
@@ -0,0 +1,29 @@
+const { sq } = require('../config/dbConnect');
+const { DataTypes } = require('sequelize');
+
+
+const User = sq.define("users", {
```

Comment:
You are not allowed to use the ORM syntax to define this model. Write RAW SQL to create tables and handle other stuff. Please reach out to Doris if you don't understand what this means.

One of the goals of this project is to understand Postgres and write SQL.